src/test/mocks/archiveLoader.ts

```diff
@@ -7,6 +7,10 @@ export const book = JSON.parse(
   fs.readFileSync(path.resolve(__dirname, '../fixtures/contents/testbook1-shortid'), 'utf8')
 ) as ArchiveBook;
 
+export const validUUIDBook = JSON.parse(
```

Comment:
for what did you add this new book?

Reply:
I've looked at the other books and their ID's don't seem to be the same format as the one used for routing. I thought that by adding a book with a format that the route expects, i can test it.